docs/glossary.rst

```diff
@@ -0,0 +1,31 @@
+Glossary
+========
+
+.. glossary::
+
+    reservoir simulator
+        Simulation of reservoir fields come in many forms, but for the purposes of 
+        res2df we only consider simulators takes a :term:`deck` as input and produces
+        term`output file`s such `.UNSRMY`. This includes, OPM flow and Eclipse.
+
+    .DATA file
+        Inputs provided to reservoir simulators such as Eclipse or OPM Flow.
```

Comment:
Input provided to reservoir simulators such as Eclipse or OPM Flow. Often a :term`.DATA file` includes other :term:`include files` with the `INCLUDE` keyword.